src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -105,6 +105,7 @@ class ChatSwitcherView extends React.Component {
         ) {
             this.textInput.focus();
         }
+        this.textInput.setNativeProps({tabindex: '-1'});
```

Comment:
I'm not sure I understand the placement of this here. Why must we set the `tabIndex` of the chat switcher input to `-1` isn't the compose input the one we want to prevent from focusing?

Reply:
@marcaaron It's the same main issue, if you tap on the compose input then arrow up then the input switches to the input of the chat switcher, which is an unwanted behavior, this line disables this behavior by removing the tabindex.

Reply:
Ok, but we are just setting it on every update? This way it will never be focusable? I'm not sure that's the behavior we want. It should still be focusable if the sidebar is in view, no?

Reply:
*** This is related to another comment so same suggestion:
I think the naming isFocusable is misleading, I think the naming should be isFocusedOnMount because the intended behavior is to focus on the input on mount or not to fix a problem where the input being focused on mobile phones behind the slider when the slider is opened, therefore tabIndex has nothing to do with isFocusable, then what I suggest is the following:

- Changing the isFocusable to isFocusedOnMount
- Moving tabIndex of -1 to inside TextInputFocusable and control it with an optional prop named isSwitchable.